client/src/lib/CaptchaV2.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import {createEventDispatcher} from "svelte";
+  export let captchaKey: string;
+
+  const dispatch = createEventDispatcher<{ token: string }>();
+
+  const captchaId = 'captcha_element';
+
+  window.captchaLoaded = () => {
+    const darkTheme = window.matchMedia && window.matchMedia('(prefers-color-scheme: dark)').matches;
+    const mobileScreen = window.innerHeight > window.innerWidth;
+
+    // @ts-ignore
```

Comment:
I would suggested to always specify exactly which rule is supposed to be ignored.

Reply:
I found a way to evade it by using the window reference. That will work!

Thanks for this input